package com.google.android.gms.internal.ads;

import javax.annotation.Nullable;

/* JADX INFO: compiled from: com.google.android.gms:play-services-ads@@23.6.0 */
/* JADX INFO: loaded from: classes2.dex */
public final class zzgeb {
    private boolean zza;

    @Nullable
    private final zzgek zzd;
    private final zzgdz zzb = zzgdz.zza;
    private zzgec zze = null;

    @Nullable
    private zzged zzf = null;

    @Nullable
    private final zzgdx zzc = null;

    /* synthetic */ zzgeb(zzgek zzgekVar, zzgef zzgefVar) {
        this.zzd = zzgekVar;
    }

    static /* bridge */ /* synthetic */ zzgdx zza(zzgeb zzgebVar) {
        zzgdx zzgdxVar = zzgebVar.zzc;
        return null;
    }

    public final zzgeb zzc() {
        zzged zzgedVar = this.zzf;
        if (zzgedVar != null) {
            zzgedVar.zzd();
        }
        this.zza = true;
        return this;
    }

    public final zzgeb zzd() {
        this.zze = zzgec.zza;
        return this;
    }
}
